src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Instead of using `toString()` to get the string representation of the enum constant, use the `name()` method. This is recommended because `name()` is specifically designed for this purpose and ensures that you always get the correct string representation of the enum constant.